### PR TITLE
Use python3 as the default python version in Spark sessions.

### PR DESCRIPTION
**Pull request by corivera**
Fixes https://github.com/microsoft/azuredatastudio/issues/7315

### CODE REVIEW

File: extensions/notebook/src/jupyter/jupyterSessionManager.ts [hunk 1]
```diff
@@ -266,6 +267,11 @@ export class JupyterSession implements nb.ISession {
 				code: doNotCallChangeEndpointParams
 			}, true);
 			await future.done;
+
+			future = this.sessionImpl.kernel.requestExecute({
+				code: `%%configure -f${EOL}{"conf": {"spark.pyspark.python": "python3"}}`
```
Comment (chlafreniere) [inline extensions/notebook/src/jupyter/jupyterSessionManager.ts]: I'm fine with this for now, but we may want to consider exposing this via a setting.

Comment (corivera) [reply extensions/notebook/src/jupyter/jupyterSessionManager.ts]: Python 2 reaches EOL in January, so we wouldn't get much use out of it.

@kevcunnane Your thoughts here?

Comment (corivera) [reply extensions/notebook/src/jupyter/jupyterSessionManager.ts]: Also Spark is dropping support for Python 2. https://spark.apache.org/news/plan-for-dropping-python-2-support.html

Comment (kevcunnane) [reply extensions/notebook/src/jupyter/jupyterSessionManager.ts]: I'm OK hard coding given the EOL decision here. If we get a user request to support Python2 for a different target in the future we can look into a setting, but for now this is a good solution.

**Pull request merged**